src/sass/sections/nav-menu.scss

```diff
@@ -0,0 +1,92 @@
+// Nav meny
+.sec-cont {
+  width: 100%;
+  height: 5rem;
+  margin: 0 auto;
+  max-width: 1000px;
+  padding: 0 5%;
+
+  .nav {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+
+    .movies-nav {
+      list-style-type: none;
+      display: inline;
+    }
+
+    .movies-nav-list {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      margin-top: 2vh;
+
+      .movies-nav-element {
+        width: fit-content;
+        text-decoration: none;
+        list-style-type: none;
+        border: 0.1rem solid black;
+        text-align: center;
+        padding: 0.45rem;
+        margin-left: 1rem;
+        border-radius: 0.5rem;
+        background-color: $second-color;
+        margin-bottom: 0.07rem;
+        position: relative;
+        font-size: larger;
```

Comment:
Just a small thing, but maybe order properties so it's easy to find. For example all margin next to each other and so on. Maybe you could use just margin instead of margin-left and margin-bottom. Just something we could thing about.

Reply:
Okej. No problem. I changed that to just margin.